src/app/organizations/registerOrganization/requireAccountsModal/require-accounts-modal.component.html

```diff
@@ -0,0 +1,12 @@
+<h1 mat-dialog-title>User Verification</h1>
+<p>
+  For verification purposes, you are required to have at least two external accounts linked to your Dockstore account. However, it's
+  preferrable to have more than two because Dockstore curators will use that information to verify that your organization request is
+  legitimate.
+</p>
+<p>You have {{ numLinkedAccounts }} account(s) linked.</p>
```

Comment:
I'm a fan of figuring out whether you need the parens or not. :) One of my pet peeves in DOS (and maybe still Windows?).

I think Angular has a pipe for this.